.travis.yml

```diff
@@ -11,11 +11,32 @@ matrix:
     - go: master
 
 install:
+  - sudo apt-get update -qq && sudo apt-get install -y -qq apache2-utils php5
   # ensure that vendor/ is in sync with code and Gopkg.*
   - curl https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 -L -o ~/dep && chmod +x ~/dep
   - rm -fr vendor/
   - ~/dep ensure -v -vendor-only
   - git diff --exit-code
 
 script:
-  go install -v ./cmd/...
+  - git status
+  - go build -v ./cmd/pmm-configure
+  - go build -v ./cmd/pmm-configurator
+  - go test -coverpkg="github.com/percona/pmm-manage/..." -c -tags testrunmain ./cmd/pmm-configurator
+  - |
+    sed -i "/ssh-key-path/assh-key-owner: $USER" tests/sandbox/config.yml
+  - PATH=$PATH:./tests/sandbox ./pmm-configurator.test -test.run "^TestRunMain$" -test.coverprofile=coverage.txt >/dev/null 2>&1 &
+  - sleep 1
```

Comment:
This could be one day made in more deterministic way e.g. https://github.com/percona/qan-agent/blob/master/.travis.yml#L64-L65 but it's fine for now.

Reply:
yep, it is possible to add loop here, but don't want to overkill things